src/appleseed/renderer/kernel/shading/closures.cpp

```diff
@@ -107,7 +107,6 @@ namespace
 
     OSL::ustring standard_dipole_profile_name("standard_dipole");
     OSL::ustring better_dipole_profile_name("better_dipole");
-    OSL::ustring directional_dipole_profile_name("directional_dipole");
```

Comment:
Why not use the `#define` you introduced earlier? (Even though I think we should not introduce this define yet, and similarly we shouldn't remove the directional dipole from OSL yet.)